lib/src/useCountdown/useCountdown.ts

```diff
@@ -95,8 +91,7 @@ function useCountdown(
   // default values
   intervalMs = intervalMs ?? 1000
   isIncrement = isIncrement ?? false
-  countStop = countStop ?? 0
-  isIncrement = isIncrement ?? false
+  countStop = (countStop ?? 0) as number
```

Comment:
Umm.. `countStop` is not actually a `number` (this got me for a while) because in the old version there was no such thing. So later on, in line 129 (old version) I had to skip calling `stopCountdown()` for the old version.

Reply:
On a second look, my syntax seems to be wrong because this would never assign `undefined` to `countStop` yet the type hint of VSCode deduced `countStop: undefined | number` because later `countStop` was used in another scope.  Maybe we keep it your way, changing the behaviour of the hook and the countdown would stop at 0.

Reply:
Good catch, I've tried to make this variable a number, but it can't. In the hook option, we indeed have `countStop: undefined | number`. I forgot to remove it to come back to `countStop = countStop ?? 0`. I will fix it, thanks!

Reply:
Seems you misunderstood me but what you did was correct. The `countStop: undefined | number` would use the number `0` as a default value. What I was trying to do was to NOT add a `countStop` for the old hook since they might be designed not to stop. It seems that we should pass a `countStop = -Infinity` when `isDeprecated`.